pkg/webui/console/containers/gateway-general-information-panel/index.js

```diff
@@ -0,0 +1,169 @@
+// Copyright © 2024 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useCallback } from 'react'
+import { useSelector } from 'react-redux'
+import { useParams } from 'react-router-dom'
+import { defineMessages } from 'react-intl'
+
+import Panel from '@ttn-lw/components/panel'
+import DataSheet from '@ttn-lw/components/data-sheet'
+import Icon, { IconCircleCheck, IconExclamationCircle } from '@ttn-lw/components/icon'
+
+import Message from '@ttn-lw/lib/components/message'
+import DateTime from '@ttn-lw/lib/components/date-time'
+
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+
+import { selectSelectedGateway } from '@console/store/selectors/gateways'
+import { selectGsFrequencyPlans } from '@console/store/selectors/configuration'
+
+import style from './gateway-general-information-panel.styl'
+
+const m = defineMessages({
```

Comment:
Aren't these now duplicates of the gateway settings tooltips?

Reply:
No, because it's not exactly the same message. These have a "When enabled" beginning, that the general settings messages don't have